.github/CODEOWNERS

```
@@ -0,0 +1,8 @@
+# Docs for how this file works: https://help.github.com/en/articles/about-code-owners
+
+# It’s not clear which team at Funding Circle should be responsible for this repo, but I created it,
+# so it’s incumbent on me to take responsibility. I’m adding Sasha as well, just as a backup.
+* avi.flax@fundingcircle.com sasha.gerrand@fundingcircle.com
```

Suggest widening the ownership to enable others to contribute.
```suggestion
* @FundingCircle/engineers
```